cypress/support/pages/admin/pageAdministration.js

```diff
@@ -538,13 +531,6 @@ class Management {
 
 	clickOnSchoolAdministrationInSideMenu() {
 		cy.get(Management.#schoolAdministrationNavigationSubMenu).first().click();
-		cy.get("body").then((body) => {
```

Comment:
if old school page is going to be removed, then line #541 to #547 should be removed in this method?

Reply:
yes, it will be removed in that case